### PR TITLE
Add Backward Compatibility for Assistant API v1

### PR DESCRIPTION
**Pull request by alperozturk96**
- [x] Tests written, or not not needed

https://github.com/user-attachments/assets/f64a4764-433b-4164-a16b-4dc26072a192

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,7 +10,7 @@
  */
 buildscript {
     ext {
-        androidLibraryVersion ="f174c1ee78"
+        androidLibraryVersion ="74d072cdf3"
```
Comment (tobiasKaminsky) [inline build.gradle:13]: I merged library PR, so I think it is better to use the merge commit 638326e14bc234ca3285852af461d41e6e08d1aa
as it then also uses latest commit to library.

File: .idea/inspectionProfiles/ktlint.xml [hunk 2]
```diff
@@ -4,6 +4,34 @@
     <inspection_tool class="AutoCloseableResource" enabled="true" level="WARNING" enabled_by_default="true">
       <option name="METHOD_MATCHER_CONFIG" value="java.util.Formatter,format,java.io.Writer,append,com.google.common.base.Preconditions,checkNotNull,org.hibernate.Session,close,java.io.PrintWriter,printf,java.io.PrintStream,printf,java.nio.channels.FileChannel,position,okhttp3.Call,execute" />
     </inspection_tool>
+    <inspection_tool class="ComposePreviewDimensionRespectsLimit" enabled="true" level="WARNING" enabled_by_default="true">
```
Comment (tobiasKaminsky) [inline .idea/inspectionProfiles/ktlint.xml:7]: Is this needed for this PR, else please split to another PR.
smaller PR, with only one topic are easier to review, less discussion, faster to merge :)

Comment (alperozturk96) [reply .idea/inspectionProfiles/ktlint.xml:7]: I didn't do anything extra. Android Studio added automatically. I have to investigate it to find out it's necessary or not.

**Review by tobiasKaminsky — APPROVED** [review]
